package.json

```diff
@@ -9,12 +9,12 @@
     "custom-login-server": "npm start --prefix custom-login/",
     "test:custom-login": "export TEST_TYPE=implicit && protractor okta-oidc-tck/e2e-tests/custom-login/conf.js",
     "resource-server": "mvn -Dokta.oauth2.issuer=$ISSUER -f samples-java-spring-mvc/resource-server/pom.xml",
-    "test:e2e": "export TEST_TYPE=implicit && npm run get-oidc-tck && npm run test:okta-hosted-login && npm run test:custom-login",
-    "get-oidc-tck": "[ ! -d okta-oidc-tck/ ] && git clone https://github.com/okta/okta-oidc-tck.git || echo \"TCK already cloned\"",
+    "test:e2e": "export TEST_TYPE=implicit && npm run test:okta-hosted-login && npm run test:custom-login",
     "pretest": "webdriver-manager update --gecko false",
     "postinstall": "npm run install-custom-login && npm run install-okta-hosted-login",
     "install-custom-login": "cd custom-login && npm install",
-    "install-okta-hosted-login": "cd okta-hosted-login && npm install"
+    "install-okta-hosted-login": "cd okta-hosted-login && npm install",
+    "test-windows": ".\\scripts\\runTests.sh"
```

Comment:
Since this will *only* get executed on Windows, would it make sense to put this all under `npm test`:

```
test: ".\\scripts\\runTests.sh || npm run test:e2e"
```

Reply:
That makes sense. Let me test it out.

Reply:
> **NOTE:** If you want to run individual tests such as `npm run test:okta-hosted-login`, you will need to update the enviroment by running the following node script
>
> ```
> node scripts/setup-env.js
> ```